<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="status">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(66, 118, 255) ;
border-color: rgb(255, 101, 16);
color : white;</string>
         </property>
         <property name="text">
          <string>Status : Uknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="score">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 100, 16);
color:white;
text-align: center;</string>
         </property>
         <property name="text">
          <string>Score : 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="username">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 101, 16);
color : white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="h_score">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(68, 114, 255);</string>
         </property>
         <property name="text">
          <string>High score : 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="MyGLWidget" name="openGLWidget">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Cam" name="imageLabel_">
         <property name="styleSheet">
          <string notr="true">background-color:bacrgb(0, 0, 0)</string>
         </property>
         <property name="text">
          <string>Camera</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="match">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 33, 29);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 112, 15);</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="giveupButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(68, 114, 255);</string>
         </property>
         <property name="text">
          <string>Give up</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">myglwidget.h</header>
  </customwidget>
  <customwidget>
   <class>Cam</class>
   <extends>QLabel</extends>
   <header>cam.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
